app/models.py

```diff
@@ -2838,6 +2838,15 @@ class UserFramework(db.Model):
     framework = db.relationship(Framework, lazy='joined', innerjoin=True)
 
 
+class BriefAssessor(db.Model):
+    __tablename__ = 'brief_assessor'
+
+    id = db.Column(db.Integer, primary_key=True)
+    brief_id = db.Column(db.Integer, db.ForeignKey('brief.id'), nullable=False)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'))
+    email_address = db.Column(db.String)
```

Comment:
can we not use the email address from user?

Reply:
tricky one - email address can be added as assessor even if they are not a valid user